.babelrc

```diff
@@ -13,7 +13,10 @@
 
     "env": {
         "test": {
-            "plugins": ["transform-es2015-modules-commonjs"]
+            "plugins": [
+                "transform-class-properties",
```

Comment:
you're using `transform-class-properties` in client code and not just for testing so you should add as a  regular plugin: https://github.com/benmvp/react-composite-events/blob/master/package.json#L58

Reply:
Oh, let's talk about it on person, we have this plugin in our babelLoader and here.

Reply:
Yeah I did notice that down below. It'd be nice if they could share the same configuration right?